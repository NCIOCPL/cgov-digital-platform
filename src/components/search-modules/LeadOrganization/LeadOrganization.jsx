import React from 'react';
import PropTypes from 'prop-types';
import Fieldset from '../../atomic/Fieldset';
import './LeadOrganization.scss';

const LeadOrganization = () => {
  return (
    <Fieldset
      id="location"
      legend="Location"
      helpUrl="https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/help#leadorganization">
      <input type="text" />
    </Fieldset>
  );
};

LeadOrganization.propTypes = {
  sampleProperty: PropTypes.string
};

LeadOrganization.defaultProps = {
  sampleProperty: 'LeadOrganization'
};

export default LeadOrganization;
