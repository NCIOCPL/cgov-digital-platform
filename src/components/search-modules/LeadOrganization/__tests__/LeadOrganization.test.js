import React from 'react';
import { render, shallow, mount } from 'enzyme';
import {
  resetDom,
  setupDom
} from '../../../../utilities/testHelpers';
import LeadOrganization from '../LeadOrganization';

const defaultProps = {
}

const setup = (enzymeMethod = render, props = {}) => {
  const component = enzymeMethod(
    <LeadOrganization {...props} />
  );
  return component;
};

describe('LeadOrganization', () => {
  describe('Render', () => {
    let component = null;

    afterEach(() => {
      resetDom();
    });

    it('renders without error', () => {
      component = setup(render, defaultProps);
      expect(component).toMatchSnapshot();
    });
  });

});
