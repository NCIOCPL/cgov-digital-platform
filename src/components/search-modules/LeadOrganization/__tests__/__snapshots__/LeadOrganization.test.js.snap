// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeadOrganization Render renders without error 1`] = `
<fieldset
  class="cts-fieldset"
  id="fieldset--location"
>
  <legend
    class="cts-fieldset__legend"
  >
    <span>
      Location
    </span>
  </legend>
  <a
    aria-label="Help"
    class="text-icon-help"
    href="https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/help#leadorganization"
    rel="noopener noreferrer"
    target="_blank"
  >
    ?
  </a>
  <div
    class="cts-fieldset__body"
  >
    <input
      type="text"
    />
  </div>
</fieldset>
`;
